build.rs

```diff
@@ -4,11 +4,14 @@ extern crate gcc;
 // a helper library for octh with c exports
 fn libocthc() {
     // https://github.com/servo/rust-bindgen/blob/master/bindgen-integration/build.rs
+    // https://github.com/alexcrichton/gcc-rs/blob/master/src/lib.rs
     gcc::Config::new()
+        .compiler("C:/Octave/Octave-4.2.1/bin/gcc.exe")
         .cpp(true)
         .file("src/octhc.cc")
         .include("C:/Octave/Octave-4.2.1/include/octave-4.2.1/octave")
-        .flag("-std=c++11")
+        // .flag("-std=c++11")
+        .flag("-std=gnu++11")
```

Comment:
enables gnu extensions
https://stackoverflow.com/questions/10613126/what-are-the-differences-between-std-c11-and-std-gnu11